src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -40,10 +40,10 @@ public function __construct(ContainerBuilder $container, FileLocatorInterface $l
     /**
      * Registers a set of classes as services using PSR-4 for discovery.
      *
-     * @param Definition $prototype A definition to use as template
-     * @param string     $namespace The namespace prefix of classes in the scanned directory
-     * @param string     $resource  The directory to look for classes, glob-patterns allowed
-     * @param string     $exclude   A globed path of files to exclude
+     * @param Definition      $prototype A definition to use as template
+     * @param string          $namespace The namespace prefix of classes in the scanned directory
+     * @param string          $resource  The directory to look for classes, glob-patterns allowed
+     * @param string|string[] $exclude   A globed path of files to exclude or an array of globed paths of files to exclude
```

Comment:
right type should actually be `string|string[]|null` (the phpdoc should be `string|null` in 3.4)